docs/investigations/storage/managed-db/managed-dbs.md

```diff
@@ -0,0 +1,108 @@
+# Managed DBs Evaluation
+
+Following document describes the evaluation of managed databases that can be used as a
+persistence layer for Compass. 
+
+## Requirements
+To find out the best storage solution for Compass, we defined the following requirements: 
+1. Fully managed solution - we don't want to spend time on managing DB.
+2. Extensible "schema" that allow searching by specifying JSON Path. 
+We plan to allow clients storing metadata for Runtime or Application as a JSON object in labels or annotations. 
+3. Easy local development.
+4. Have an alternative that can be installed in k8s cluster.
```

Comment:
What's the difference between point 3 and 4?

Reply:
Local development can be a Docker image run on your local machine, so it does not require k8s at all.